docs/protocol/pulsar.mdx

```diff
@@ -302,57 +297,58 @@ Ubuntu:
 
 ```bash
 
-./subspace-cli-ubuntu-x86_64-skylake-v0.5.3-alpha-2 init
+./pulsar-ubuntu-x86_64-skylake-v0.6.0-alpha  init
 
 ```
 Ubuntu Executable (aarch64):
 
 ```bash
 
-./subspace-cli-ubuntu-aarch64-v0.5.3-alpha-2 init
+./pulsar-ubuntu-aarch64-v0.6.0-alpha  init
 
 ```
 
 </TabItem>
 </Tabs>
 
-This will prompt you to setup your CLI configurations to begin farming. You should see a similar prompt like so (some info might be different than shown here w.r.t your OS):
+This will prompt you to setup your Pulsar configurations to begin farming. You should see a similar prompt like so (some info might be different than shown here w.r.t your OS):
 
 ```bash
-version: 0.5.0
+version: 0.6.0
 
 Configuration creation process has started...
+Do you have an existing farmer/reward address? [y/n]: y
 Enter your farmer/reward address: REDACTED_ADDRESS
-Enter your node name to be identified on the network(defaults to `username`, press enter to use the default):
-Specify a path for storing plot files (press enter to use the default: `"/home/username/.local/share/subspace-cli/plots"`):
-Specify a path for storing node files (press enter to use the default: `"/home/username/.local/share/subspace-cli/node"`):
-Specify a plot size (defaults to `1000.0 MB`, press enter to use the default):
-Specify the chain to farm. Available options are: [Gemini3e, Dev, DevNet].
+Enter your node name to be identified on the network(defaults to `subspace`, press enter to use the default):
+Specify a path for storing plot files (press enter to use the default: `"/home/username/.local/share/pulsar/plots"`):
+Specify a path for storing node files (press enter to use the default: `"/home/username/.local/share/pulsar/node"`):
+Specify a plot size (defaults to `2.0 GB`, press enter to use the default):
+Specify the chain to farm. Available options are: [Gemini3f, Dev, DevNet].
```

Comment:
default version on the next line also needs to be changed Gemini3e -> Gemini3f

Reply:
Please feel free to do it directly here @extSunset!

Reply:
On the second though, let's keep this focused on Pulsar and address unrelated changes in separate PRs @extSunset. Thanks!